react/PopupOpener/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// customized function to center a popup window
+// source https://stackoverflow.com/a/16861050
+export function popupCenter(url, title, w, h) {
+  /* global screen */
+  // Fixes dual-screen position
+  //                      Most browsers      Firefox
+  var dualScreenLeft = window.screenLeft || screen.left
+  var dualScreenTop = window.screenTop || screen.top
```

Comment:
Not blocking : screenLeft and screenTop seem more appropriate here.

Reply:
Removed to use directly `window.screenX` and `window.screenY`